code/modules/cm_marines/super_secret.dm

```diff
@@ -56,12 +56,12 @@
 							for(var/mob/living/carbon/human/H in range(12))
 								if(!H || isnull(H) || H.stat) continue
 								to_chat(H, "<font size=8 color=red><b>NO<br><br>BUGS</b></font>")
-							new /obj/item/weapon/gun/launcher/rocket/nobugs(get_turf(src.loc))
```

Comment:
unrelated to the PR, should be removed.

Reply:
Ideally this should be on its own PR, for sure.

Reply:
I'll do better in the future.